### ISSUE TITLE
[0.1.10] Ticking entity crashes from The Other

### ISSUE THREAD

**Issue by Brother-Blue** [AllTheMods/ATM-9#543]
Something about going into The Other with an activated jetpack or activating your jetpack while in The Other causes the server to seize up:

```log
java.lang.Error: ServerHangWatchdog detected that a single server tick took 180.00 seconds (should be max 0.05)
	at net.minecraft.server.dedicated.ServerWatchdog.run(ServerWatchdog.java:43) ~[server-1.20.1-20230612.114412-srg.jar%23765!/:?] {re:classloading}
	at java.lang.Thread.run(Thread.java:833) ~[?:?] {re:mixin}
```

This only happened when one of my players entered The Other with the jetpack activated or was already in The Other with it activated - which hardlocked the server as they could never connect in time to tp out. Once I had changed their playerdata to move them back into the overworld the server responded just fine, and when they go back into The Other with the jetpack deactivated or unequipped the server no longer crashes.

Full crash report:
[crash-2023-10-07_20.40.31-server.txt](https://github.com/AllTheMods/ATM-9/files/12840214/crash-2023-10-07_20.40.31-server.txt)

**Comment by Brother-Blue** [comment]
This just happened again but in a completely different way this time; there was no jetpack on the player.

```log
java.lang.ClassCastException: class net.minecraft.world.entity.monster.piglin.Piglin cannot be cast to class net.minecraft.server.level.ServerPlayer (net.minecraft.world.entity.monster.piglin.Piglin and net.minecraft.server.level.ServerPlayer are in module minecraft@1.20.1 of loader 'TRANSFORMER' @5b1f0f26)
	at com.kyanite.deeperdarker.content.enchantments.CatalysisEnchantment.m_7677_(CatalysisEnchantment.java:33) ~[deeperdarker-forge-1.20.1-1.2.0.jar%23530!/:1.2.0] {re:classloading}
	at net.minecraft.world.item.enchantment.EnchantmentHelper.m_44826_(EnchantmentHelper.java:194) ~[server-1.20.1-20230612.114412-srg.jar%23765!/:?] {re:mixin,re:classloading,pl:mixin:APP:apotheosis.mixins.json:EnchantmentHelperMixin,pl:mixin:APP:ae2.mixins.json:EnchantmentHelperMixin,pl:mixin:APP:mixins.artifacts.common.json:item.wearable.charmofsinking.EnchantmentHelperMixin,pl:mixin:A}
	at net.minecraft.world.item.enchantment.EnchantmentHelper.m_44850_(EnchantmentHelper.java:132) ~[server-1.20.1-20230612.114412-srg.jar%23765!/:?] {re:mixin,re:classloading,pl:mixin:APP:apotheosis.mixins.json:EnchantmentHelperMixin,pl:mixin:APP:ae2.mixins.json:EnchantmentHelperMixin,pl:mixin:APP:mixins.artifacts.common.json:item.wearable.charmofsinking.EnchantmentHelperMixin,pl:mixin:A}
	at net.minecraft.world.item.enchantment.EnchantmentHelper.m_44853_(EnchantmentHelper.java:151) ~[server-1.20.1-20230612.114412-srg.jar%23765!/:?] {re:mixin,re:classloading,pl:mixin:APP:apotheosis.mixins.json:EnchantmentHelperMixin,pl:mixin:APP:ae2.mixins.json:EnchantmentHelperMixin,pl:mixin:APP:mixins.artifacts.common.json:item.wearable.charmofsinking.EnchantmentHelperMixin,pl:mixin:A}
	at net.minecraft.world.item.enchantment.EnchantmentHelper.m_44896_(EnchantmentHelper.java:197) ~[server-1.20.1-20230612.114412-srg.jar%23765!/:?] {re:mixin,re:classloading,pl:mixin:APP:apotheosis.mixins.json:EnchantmentHelperMixin,pl:mixin:APP:ae2.mixins.json:EnchantmentHelperMixin,pl:mixin:APP:mixins.artifacts.common.json:item.wearable.charmofsinking.EnchantmentHelperMixin,pl:mixin:A}
	at net.minecraft.world.entity.Entity.m_19970_(Entity.java:2830) ~[server-1.20.1-20230612.114412-srg.jar%23765!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:create.mixins.json:EntityMixin,pl:mixin:APP:kubejs-forge.mixins.json:EntityMixin,pl:mixin:APP:kubejs-common.mixins.json:EntityMixin,pl:mixin:APP:ad_astra_giselle_addon.mixin.common.json:minecraft.EntityMixin,pl:mixin:APP:shrink.mixins.json:MixinEntity,pl:mixin:APP:balm.mixins.json:EntityMixin,pl:mixin:APP:mixins.shetiphiancore.json:SPC_ThinBlockHandler$_Entity,pl:mixin:APP:supplementaries-common.mixins.json:EntityMixin,pl:mixin:APP:curios.mixins.json:AccessorEntity,pl:mixin:APP:attributeslib.mixins.json:EntityMixin,pl:mixin:APP:tombstone.mixins.json:EntityMixin,pl:mixin:APP:l2library.mixins.json:EntityMixin,pl:mixin:APP:sliceanddice.mixins.json:EntityMixin,pl:mixin:APP:mixins.pneumaticcraft.json:accessors.EntityAccess,pl:mixin:APP:corgilib.mixins.json:MixinEntity,pl:mixin:APP:ad_astra-common.mixins.json:EntityInvoker,pl:mixin:APP:ad_astra-common.mixins.json:EntityMixin,pl:mixin:APP:netherportalfix.mixins.json:EntityAccessor,pl:mixin:APP:bookshelf.common.mixins.json:accessors.entity.AccessorEntity,pl:mixin:APP:railways-common.mixins.json:conductor_possession.MixinEntity,pl:mixin:APP:twilightdelight.mixins.json:EntityMixin,pl:mixin:APP:blueflame.mixin.json:MixinEntity,pl:mixin:APP:mixins.artifacts.common.json:item.umbrella.EntityMixin,pl:mixin:APP:mixins.artifacts.common.json:item.wearable.aquadashers.EntityMixin,pl:mixin:APP:mixins.irons_spellbooks.json:EntityMixin,pl:mixin:APP:expandability.mixins.json:swimming.EntityMixin,pl:mixin:APP:expandability-common.mixins.json:fluidcollision.EntityMixin,pl:mixin:APP:expandability-common.mixins.json:swimming.EntityMixin,pl:mixin:APP:create.mixins.json:ContraptionDriverInteractMixin,pl:mixin:APP:create.mixins.json:accessor.EntityAccessor,pl:mixin:APP:structure_gel.mixins.json:EntityMixin,pl:mixin:A}
	at net.minecraft.world.entity.Mob.m_7327_(Mob.java:1422) ~[server-1.20.1-20230612.114412-srg.jar%23765!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:tombstone.mixins.json:MobMixin,pl:mixin:APP:mixins.pneumaticcraft.json:accessors.MobAccess,pl:mixin:APP:ars_nouveau.mixins.json:jar.MobAccessorMixin,pl:mixin:APP:bookshelf.common.mixins.json:accessors.entity.AccessorMob,pl:mixin:APP:mixins.letmedespawn.json:MobMixin,pl:mixin:APP:mixins.artifacts.common.json:accessors.MobAccessor,pl:mixin:APP:moonlight-common.mixins.json:EntityMixin,pl:mixin:A}
	at net.minecraft.world.entity.ai.behavior.MeleeAttack.m_257216_(MeleeAttack.java:27) ~[server-1.20.1-20230612.114412-srg.jar%23765!/:?] {re:classloading}
	at net.minecraft.world.entity.ai.behavior.declarative.BehaviorBuilder$1.m_257808_(BehaviorBuilder.java:53) ~[server-1.20.1-20230612.114412-srg.jar%23765!/:?] {re:classloading}
	at net.minecraft.world.entity.ai.behavior.OneShot.m_22554_(OneShot.java:20) ~[server-1.20.1-20230612.114412-srg.jar%23765!/:?] {re:classloading}
	at net.minecraft.world.entity.ai.Brain.m_21957_(Brain.java:533) ~[server-1.20.1-20230612.114412-srg.jar%23765!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
	at net.minecraft.world.entity.ai.Brain.m_21865_(Brain.java:492) ~[server-1.20.1-20230612.114412-srg.jar%23765!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
	at net.minecraft.world.entity.monster.piglin.Piglin.m_8024_(Piglin.java:254) ~[server-1.20.1-20230612.114412-srg.jar%23765!/:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.world.entity.Mob.m_6140_(Mob.java:768) ~[server-1.20.1-20230612.114412-srg.jar%23765!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:tombstone.mixins.json:MobMixin,pl:mixin:APP:mixins.pneumaticcraft.json:accessors.MobAccess,pl:mixin:APP:ars_nouveau.mixins.json:jar.MobAccessorMixin,pl:mixin:APP:bookshelf.common.mixins.json:accessors.entity.AccessorMob,pl:mixin:APP:mixins.letmedespawn.json:MobMixin,pl:mixin:APP:mixins.artifacts.common.json:accessors.MobAccessor,pl:mixin:APP:moonlight-common.mixins.json:EntityMixin,pl:mixin:A}
	at net.minecraft.world.entity.LivingEntity.m_8107_(LivingEntity.java:2542) ~[server-1.20.1-20230612.114412-srg.jar%23765!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:kubejs-common.mixins.json:LivingEntityMixin,pl:mixin:APP:ad_astra_giselle_addon.mixin.common.json:minecraft.LivingEntityMixin,pl:mixin:APP:apotheosis.mixins.json:LivingEntityInvoker,pl:mixin:APP:apotheosis.mixins.json:LivingEntityMixin,pl:mixin:APP:apotheosis.mixins.json:MHFMixinLivingEntity,pl:mixin:APP:mixins.shetiphiancore.json:SPC_ThinBlockHandler$_LivingEntity,pl:mixin:APP:supplementaries-common.mixins.json:LivingEntityMixin,pl:mixin:APP:curios.mixins.json:MixinLivingEntity,pl:mixin:APP:attributeslib.mixins.json:LivingEntityMixin,pl:mixin:APP:tombstone.mixins.json:LivingEntityMixin,pl:mixin:APP:caelus.mixins.json:MixinLivingEntity,pl:mixin:APP:ars_nouveau.mixins.json:elytra.MixinLivingEntity,pl:mixin:APP:ars_nouveau.mixins.json:perks.PerkLivingEntity,pl:mixin:APP:ad_astra-common.mixins.json:LivingEntityMixin,pl:mixin:APP:ad_astra-common.mixins.json:gravity.LivingEntityGravityMixin,pl:mixin:APP:blue_skies.mixins.json:LivingEntityMixin,pl:mixin:APP:netherportalfix.mixins.json:LivingEntityAccessor,pl:mixin:APP:citadel.mixins.json:LivingEntityMixin,pl:mixin:APP:bookshelf.common.mixins.json:accessors.entity.AccessorLivingEntity,pl:mixin:APP:bookshelf.common.mixins.json:patches.entity.MixinLivingEntity,pl:mixin:APP:cataclysm.mixins.json:LivingEntityMixin,pl:mixin:APP:voidtotem.mixins.json:LivingEntityMixin,pl:mixin:APP:caupona.mixins.json:LivingEntityMixin,pl:mixin:APP:mixins.artifacts.common.json:item.hurtsound.LivingEntityMixin,pl:mixin:APP:betterdeserttemples.mixins.json:PharaohKilledMixin,pl:mixin:APP:mixins.enderio.json:LivingEntityMixin,pl:mixin:APP:mixins.irons_spellbooks.json:LivingEntityMixin,pl:mixin:APP:enhancedcelestials.mixins.json:MixinLivingEntity,pl:mixin:APP:charmofundying.mixins.json:MixinLivingEntity,pl:mixin:APP:expandability.mixins.json:swimming.LivingEntityMixin,pl:mixin:APP:expandability-common.mixins.json:swimming.LivingEntityMixin,pl:mixin:APP:create.mixins.json:CustomItemUseEffectsMixin,pl:mixin:APP:create.mixins.json:LavaSwimmingMixin,pl:mixin:APP:create.mixins.json:accessor.LivingEntityAccessor,pl:mixin:A}
	at net.minecraft.world.entity.Mob.m_8107_(Mob.java:536) ~[server-1.20.1-20230612.114412-srg.jar%23765!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:tombstone.mixins.json:MobMixin,pl:mixin:APP:mixins.pneumaticcraft.json:accessors.MobAccess,pl:mixin:APP:ars_nouveau.mixins.json:jar.MobAccessorMixin,pl:mixin:APP:bookshelf.common.mixins.json:accessors.entity.AccessorMob,pl:mixin:APP:mixins.letmedespawn.json:MobMixin,pl:mixin:APP:mixins.artifacts.common.json:accessors.MobAccessor,pl:mixin:APP:moonlight-common.mixins.json:EntityMixin,pl:mixin:A}
	at net.minecraft.world.entity.monster.Monster.m_8107_(Monster.java:42) ~[server-1.20.1-20230612.114412-srg.jar%23765!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:tombstone.mixins.json:MonsterMixin,pl:mixin:A}
	at net.minecraft.world.entity.LivingEntity.m_8119_(LivingEntity.java:2292) ~[server-1.20.1-20230612.114412-srg.jar%23765!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:kubejs-common.mixins.json:LivingEntityMixin,pl:mixin:APP:ad_astra_giselle_addon.mixin.common.json:minecraft.LivingEntityMixin,pl:mixin:APP:apotheosis.mixins.json:LivingEntityInvoker,pl:mixin:APP:apotheosis.mixins.json:LivingEntityMixin,pl:mixin:APP:apotheosis.mixins.json:MHFMixinLivingEntity,pl:mixin:APP:mixins.shetiphiancore.json:SPC_ThinBlockHandler$_LivingEntity,pl:mixin:APP:supplementaries-common.mixins.json:LivingEntityMixin,pl:mixin:APP:curios.mixins.json:MixinLivingEntity,pl:mixin:APP:attributeslib.mixins.json:LivingEntityMixin,pl:mixin:APP:tombstone.mixins.json:LivingEntityMixin,pl:mixin:APP:caelus.mixins.json:MixinLivingEntity,pl:mixin:APP:ars_nouveau.mixins.json:elytra.MixinLivingEntity,pl:mixin:APP:ars_nouveau.mixins.json:perks.PerkLivingEntity,pl:mixin:APP:ad_astra-common.mixins.json:LivingEntityMixin,pl:mixin:APP:ad_astra-common.mixins.json:gravity.LivingEntityGravityMixin,pl:mixin:APP:blue_skies.mixins.json:LivingEntityMixin,pl:mixin:APP:netherportalfix.mixins.json:LivingEntityAccessor,pl:mixin:APP:citadel.mixins.json:LivingEntityMixin,pl:mixin:APP:bookshelf.common.mixins.json:accessors.entity.AccessorLivingEntity,pl:mixin:APP:bookshelf.common.mixins.json:patches.entity.MixinLivingEntity,pl:mixin:APP:cataclysm.mixins.json:LivingEntityMixin,pl:mixin:APP:voidtotem.mixins.json:LivingEntityMixin,pl:mixin:APP:caupona.mixins.json:LivingEntityMixin,pl:mixin:APP:mixins.artifacts.common.json:item.hurtsound.LivingEntityMixin,pl:mixin:APP:betterdeserttemples.mixins.json:PharaohKilledMixin,pl:mixin:APP:mixins.enderio.json:LivingEntityMixin,pl:mixin:APP:mixins.irons_spellbooks.json:LivingEntityMixin,pl:mixin:APP:enhancedcelestials.mixins.json:MixinLivingEntity,pl:mixin:APP:charmofundying.mixins.json:MixinLivingEntity,pl:mixin:APP:expandability.mixins.json:swimming.LivingEntityMixin,pl:mixin:APP:expandability-common.mixins.json:swimming.LivingEntityMixin,pl:mixin:APP:create.mixins.json:CustomItemUseEffectsMixin,pl:mixin:APP:create.mixins.json:LavaSwimmingMixin,pl:mixin:APP:create.mixins.json:accessor.LivingEntityAccessor,pl:mixin:A}
	at net.minecraft.world.entity.Mob.m_8119_(Mob.java:337) ~[server-1.20.1-20230612.114412-srg.jar%23765!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:tombstone.mixins.json:MobMixin,pl:mixin:APP:mixins.pneumaticcraft.json:accessors.MobAccess,pl:mixin:APP:ars_nouveau.mixins.json:jar.MobAccessorMixin,pl:mixin:APP:bookshelf.common.mixins.json:accessors.entity.AccessorMob,pl:mixin:APP:mixins.letmedespawn.json:MobMixin,pl:mixin:APP:mixins.artifacts.common.json:accessors.MobAccessor,pl:mixin:APP:moonlight-common.mixins.json:EntityMixin,pl:mixin:A}
	at net.minecraft.server.level.ServerLevel.track(ServerLevel.java:6669) ~[server-1.20.1-20230612.114412-srg.jar%23765!/:?] {re:mixin,pl:accesstransformer:B,xf:fml:libx:level_load,re:classloading,pl:accesstransformer:B,xf:fml:libx:level_load,pl:mixin:APP:cupboard.mixins.json:ServerAddEntityMixin,pl:mixin:APP:betterendisland.mixins.json:ServerLevelMixin,pl:mixin:APP:kubejs-common.mixins.json:ServerLevelMixin,pl:mixin:APP:modernfix-common.mixins.json:bugfix.chunk_deadlock.ServerLevelMixin,pl:mixin:APP:modernfix-common.mixins.json:perf.cache_strongholds.ServerLevelMixin,pl:mixin:APP:supplementaries-common.mixins.json:ServerLevelMixin,pl:mixin:APP:corgilib.mixins.json:MixinServerLevel,pl:mixin:APP:hyperbox.mixins.json:ServerLevelMixin,pl:mixin:APP:ad_astra-common.mixins.json:ServerLevelMixin,pl:mixin:APP:citadel.mixins.json:ServerLevelMixin,pl:mixin:APP:observable.common.json:ServerLevelMixin,pl:mixin:APP:betterdeserttemples.mixins.json:ServerLevelMixin,pl:mixin:APP:enhancedcelestials.mixins.json:MixinServerWorld,pl:mixin:APP:create.mixins.json:accessor.ServerLevelAccessor,pl:mixin:APP:betterendisland.mixins.json:EndergeticExpansionMixins,pl:mixin:A}
	at net.minecraft.server.level.ServerLevel.redirect$bop000$onTickNonPassenger(ServerLevel.java:6685) ~[server-1.20.1-20230612.114412-srg.jar%23765!/:?] {re:mixin,pl:accesstransformer:B,xf:fml:libx:level_load,re:classloading,pl:accesstransformer:B,xf:fml:libx:level_load,pl:mixin:APP:cupboard.mixins.json:ServerAddEntityMixin,pl:mixin:APP:betterendisland.mixins.json:ServerLevelMixin,pl:mixin:APP:kubejs-common.mixins.json:ServerLevelMixin,pl:mixin:APP:modernfix-common.mixins.json:bugfix.chunk_deadlock.ServerLevelMixin,pl:mixin:APP:modernfix-common.mixins.json:perf.cache_strongholds.ServerLevelMixin,pl:mixin:APP:supplementaries-common.mixins.json:ServerLevelMixin,pl:mixin:APP:corgilib.mixins.json:MixinServerLevel,pl:mixin:APP:hyperbox.mixins.json:ServerLevelMixin,pl:mixin:APP:ad_astra-common.mixins.json:ServerLevelMixin,pl:mixin:APP:citadel.mixins.json:ServerLevelMixin,pl:mixin:APP:observable.common.json:ServerLevelMixin,pl:mixin:APP:betterdeserttemples.mixins.json:ServerLevelMixin,pl:mixin:APP:enhancedcelestials.mixins.json:MixinServerWorld,pl:mixin:APP:create.mixins.json:accessor.ServerLevelAccessor,pl:mixin:APP:betterendisland.mixins.json:EndergeticExpansionMixins,pl:mixin:A}
	at net.minecraft.server.level.ServerLevel.m_8647_(ServerLevel.java:693) ~[server-1.20.1-20230612.114412-srg.jar%23765!/:?] {re:mixin,pl:accesstransformer:B,xf:fml:libx:level_load,re:classloading,pl:accesstransformer:B,xf:fml:libx:level_load,pl:mixin:APP:cupboard.mixins.json:ServerAddEntityMixin,pl:mixin:APP:betterendisland.mixins.json:ServerLevelMixin,pl:mixin:APP:kubejs-common.mixins.json:ServerLevelMixin,pl:mixin:APP:modernfix-common.mixins.json:bugfix.chunk_deadlock.ServerLevelMixin,pl:mixin:APP:modernfix-common.mixins.json:perf.cache_strongholds.ServerLevelMixin,pl:mixin:APP:supplementaries-common.mixins.json:ServerLevelMixin,pl:mixin:APP:corgilib.mixins.json:MixinServerLevel,pl:mixin:APP:hyperbox.mixins.json:ServerLevelMixin,pl:mixin:APP:ad_astra-common.mixins.json:ServerLevelMixin,pl:mixin:APP:citadel.mixins.json:ServerLevelMixin,pl:mixin:APP:observable.common.json:ServerLevelMixin,pl:mixin:APP:betterdeserttemples.mixins.json:ServerLevelMixin,pl:mixin:APP:enhancedcelestials.mixins.json:MixinServerWorld,pl:mixin:APP:create.mixins.json:accessor.ServerLevelAccessor,pl:mixin:APP:betterendisland.mixins.json:EndergeticExpansionMixins,pl:mixin:A}
	at net.minecraft.world.level.Level.m_46653_(Level.java:479) ~[server-1.20.1-20230612.114412-srg.jar%23765!/:?] {re:mixin,pl:accesstransformer:B,xf:fml:twilightforest:cloud,re:computing_frames,pl:accesstransformer:B,xf:fml:twilightforest:cloud,re:classloading,pl:accesstransformer:B,xf:fml:twilightforest:cloud,pl:mixin:APP:kubejs-common.mixins.json:LevelMixin,pl:mixin:APP:sliceanddice.mixins.json:LevelMixin,pl:mixin:APP:utilitix.mixins.json:MixinLevel,pl:mixin:APP:citadel.mixins.json:LevelMixin,pl:mixin:APP:observable.common.json:LevelMixin,pl:mixin:APP:gtceu-common.mixins.json:LevelMixin,pl:mixin:APP:enhancedcelestials.mixins.json:MixinWorld,pl:mixin:A}
	at net.minecraft.server.level.ServerLevel.m_184063_(ServerLevel.java:343) ~[server-1.20.1-20230612.114412-srg.jar%23765!/:?] {re:mixin,pl:accesstransformer:B,xf:fml:libx:level_load,re:classloading,pl:accesstransformer:B,xf:fml:libx:level_load,pl:mixin:APP:cupboard.mixins.json:ServerAddEntityMixin,pl:mixin:APP:betterendisland.mixins.json:ServerLevelMixin,pl:mixin:APP:kubejs-common.mixins.json:ServerLevelMixin,pl:mixin:APP:modernfix-common.mixins.json:bugfix.chunk_deadlock.ServerLevelMixin,pl:mixin:APP:modernfix-common.mixins.json:perf.cache_strongholds.ServerLevelMixin,pl:mixin:APP:supplementaries-common.mixins.json:ServerLevelMixin,pl:mixin:APP:corgilib.mixins.json:MixinServerLevel,pl:mixin:APP:hyperbox.mixins.json:ServerLevelMixin,pl:mixin:APP:ad_astra-common.mixins.json:ServerLevelMixin,pl:mixin:APP:citadel.mixins.json:ServerLevelMixin,pl:mixin:APP:observable.common.json:ServerLevelMixin,pl:mixin:APP:betterdeserttemples.mixins.json:ServerLevelMixin,pl:mixin:APP:enhancedcelestials.mixins.json:MixinServerWorld,pl:mixin:APP:create.mixins.json:accessor.ServerLevelAccessor,pl:mixin:APP:betterendisland.mixins.json:EndergeticExpansionMixins,pl:mixin:A}
	at net.minecraft.world.level.entity.EntityTickList.m_156910_(EntityTickList.java:54) ~[server-1.20.1-20230612.114412-srg.jar%23765!/:?] {re:classloading}
	at net.minecraft.server.level.ServerLevel.m_8793_(ServerLevel.java:323) ~[server-1.20.1-20230612.114412-srg.jar%23765!/:?] {re:mixin,pl:accesstransformer:B,xf:fml:libx:level_load,re:classloading,pl:accesstransformer:B,xf:fml:libx:level_load,pl:mixin:APP:cupboard.mixins.json:ServerAddEntityMixin,pl:mixin:APP:betterendisland.mixins.json:ServerLevelMixin,pl:mixin:APP:kubejs-common.mixins.json:ServerLevelMixin,pl:mixin:APP:modernfix-common.mixins.json:bugfix.chunk_deadlock.ServerLevelMixin,pl:mixin:APP:modernfix-common.mixins.json:perf.cache_strongholds.ServerLevelMixin,pl:mixin:APP:supplementaries-common.mixins.json:ServerLevelMixin,pl:mixin:APP:corgilib.mixins.json:MixinServerLevel,pl:mixin:APP:hyperbox.mixins.json:ServerLevelMixin,pl:mixin:APP:ad_astra-common.mixins.json:ServerLevelMixin,pl:mixin:APP:citadel.mixins.json:ServerLevelMixin,pl:mixin:APP:observable.common.json:ServerLevelMixin,pl:mixin:APP:betterdeserttemples.mixins.json:ServerLevelMixin,pl:mixin:APP:enhancedcelestials.mixins.json:MixinServerWorld,pl:mixin:APP:create.mixins.json:accessor.ServerLevelAccessor,pl:mixin:APP:betterendisland.mixins.json:EndergeticExpansionMixins,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_5703_(MinecraftServer.java:893) ~[server-1.20.1-20230612.114412-srg.jar%23765!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:kubejs-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:kubejs-common.mixins.json:inject_resources.MinecraftServerMixin,pl:mixin:APP:modernfix-common.mixins.json:perf.dedicated_reload_executor.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:ae2.mixins.json:spatial.MinecraftServerMixin,pl:mixin:APP:citadel.mixins.json:MinecraftServerMixin,pl:mixin:APP:dankstorage.mixins.json:MinecraftServerAccess,pl:mixin:A}
	at net.minecraft.server.dedicated.DedicatedServer.m_5703_(DedicatedServer.java:273) ~[server-1.20.1-20230612.114412-srg.jar%23765!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:mixins/common/nochatreports.mixins.json:server.MixinDedicatedServer,pl:mixin:APP:tombstone.mixins.json:DedicatedServerMixin,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_5705_(MinecraftServer.java:814) ~[server-1.20.1-20230612.114412-srg.jar%23765!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:kubejs-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:kubejs-common.mixins.json:inject_resources.MinecraftServerMixin,pl:mixin:APP:modernfix-common.mixins.json:perf.dedicated_reload_executor.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:ae2.mixins.json:spatial.MinecraftServerMixin,pl:mixin:APP:citadel.mixins.json:MinecraftServerMixin,pl:mixin:APP:dankstorage.mixins.json:MinecraftServerAccess,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_130011_(MinecraftServer.java:661) ~[server-1.20.1-20230612.114412-srg.jar%23765!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:kubejs-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:kubejs-common.mixins.json:inject_resources.MinecraftServerMixin,pl:mixin:APP:modernfix-common.mixins.json:perf.dedicated_reload_executor.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:ae2.mixins.json:spatial.MinecraftServerMixin,pl:mixin:APP:citadel.mixins.json:MinecraftServerMixin,pl:mixin:APP:dankstorage.mixins.json:MinecraftServerAccess,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_206580_(MinecraftServer.java:251) ~[server-1.20.1-20230612.114412-srg.jar%23765!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:kubejs-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:kubejs-common.mixins.json:inject_resources.MinecraftServerMixin,pl:mixin:APP:modernfix-common.mixins.json:perf.dedicated_reload_executor.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:ae2.mixins.json:spatial.MinecraftServerMixin,pl:mixin:APP:citadel.mixins.json:MinecraftServerMixin,pl:mixin:APP:dankstorage.mixins.json:MinecraftServerAccess,pl:mixin:A}
	at java.lang.Thread.run(Thread.java:833) ~[?:?] {re:mixin}
```

Full crash report:
[crash-2023-10-08_08.57.27-server.txt](https://github.com/AllTheMods/ATM-9/files/12840225/crash-2023-10-08_08.57.27-server.txt)

**Comment by alfredggttv** [comment]
`ServerHangWatchdog` means that the server stopped after it lagged for too long. You can change the setting in your `serverproperties` file to -1 to disable this, or put it to a much higher number.

Any chance they were wearing something that gives them invisibilty?

**Comment by alfredggttv** [comment]
I'd set it to a higher number then. If you keep it at default, it will most likely constantly crash you as you get further into the pack. Essentially it just closes the server if something takes too long to load, which doesn't necessarily mean it isn't working, just taking a little bit.

**Comment by Brother-Blue** [comment]
I'll give both extending watchdog and disabling it a go tomorrow and see if there's any major changes. Do you have any thoughts of what could be done regarding the second crash report? 

That one just happened randomly while another player was killing blazes in The Other they said.

**Comment by alfredggttv** [comment]
It has something to do with this `enchantments.CatalysisEnchantment`. Seems to be spawning on mobs and causing crashes.

**Comment by itsSilverlink** [comment]
I can confirm, being in the other makes the server crash after a while.

**Comment by Brother-Blue** [comment]
Any news if this has been fixed? Currently running on 0.2.10 and haven't had any issues so far but I'd like to close the issue if this has been resolved.

**Comment by Brother-Blue** [comment]
Haven't had any issues with this recently so I'll close the issue.
